mypy/semanal.py

```diff
@@ -219,7 +219,7 @@ class SemanticAnalyzer(NodeVisitor[None],
     errors: Errors  # Keeps track of generated errors
     plugin: Plugin  # Mypy plugin for special casing of library features
     statement: Optional[Statement] = None  # Statement/definition being analyzed
-    future_import_flags: Set[str]
+    future_import_flags: Dict[str, Set[str]]  # Module name / future imports
```

Comment:
Could these be stored in `MypyFile` instead, to avoid two dictionaries keyed with a module name (see `modules` above)?

Reply:
Done! 👍